sdk/objectanchors/Azure.MixedReality.ObjectAnchors/src/Customizations/Models/AssetConversionStatus.cs

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Core;
+
+namespace Azure.MixedReality.ObjectAnchors
+{
+    /// <summary> The AssetConversionStatus. </summary>
+    [CodeGenModel("JobStatus")]
+    public enum AssetConversionStatus
+    {
+        /// <summary> NotStarted. </summary>
+        NotStarted,
+        /// <summary> Running. </summary>
+        Running,
+        /// <summary> Succeeded. </summary>
+        Succeeded,
+        /// <summary> Failed. </summary>
+        Failed,
+        /// <summary> Cancelled. </summary>
+        Cancelled
```

Comment:
Do you really have to enumerate all of these just to rename the enumeration?

Reply:
Unfortunately... https://github.com/Azure/autorest.csharp#renaming-an-enum